src/cloudWatchLogs/constants.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const CLOUDWATCH_LOGS_SCHEME = 'awsCloudWatchLogs'
```

Comment:
Yeah, the thought process was: this constant is only relevant to Cloudwatch Logs and I didn't want to have any circular imports (since this is used in a variety of places). I can probably move this to the normal constants file.